test/html_test.js

```diff
@@ -47,13 +48,15 @@ exports.htmllint = {
       });
       run(test, {
         files: ['test/valid.html', 'test/invalid.html'],
-        absoluteFilePathsForReporter: true
+        absoluteFilePathsForReporter: true,
+        errorlevels: ['info','warning','error']
       }, expected, 'four errors from test/invalid.html');
     }
   },
   'empty': function(test) {
     run(test, {
-      files: []
+      files: [],
+	  errorlevels: ['info','warning','error']
```

Comment:
This looks like it has the wrong indent. There's also missing spaces after the commas (inside the array), also applies for all the other arrays like this.